src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -326,6 +321,59 @@ static bool matchMetadata(BSONObj& md1, BSONObj& md2)
 static bool equalMetadataValues(BSONObj& md1, BSONObj& md2)
 {
 
+
+  // Same declared type ?
+  if (getField(md1, ENT_ATTRS_MD_TYPE).type()!= getField(md2, ENT_ATTRS_MD_TYPE).type())
+  {
+    return false;
+  }
+  switch (getField(md1, ENT_ATTRS_MD_TYPE).type())
+  {
+    /* FIXME not yet
```

Comment:
Maybe a reference to the issue about this #1068 could be include in the FIXME comment.